pkg/services/baremetal/baremetal/baremetal_test.go

```diff
@@ -208,6 +234,9 @@ var _ = Describe("chooseHost", func() {
 		Hosts            []client.Object
 		HostSelector     infrav1.HostSelector
 		ExpectedHostName string
+		ExpectedReason   string
+		Swraid           int
```

Comment:
can you define a separate struct for the test case below? There is no overlap in the properties you need  expect for "Hosts" and "ExpectedHostName". I think it is cleaner if we have two structs

Reply:
done in d286906e. I use unexported field names in the new struct, since there is no need to export them. ok?